### ISSUE TITLE
Write database-backed example for kotlin

### ISSUE THREAD

**Issue by rjrjr** [square/workflow-kotlin#769]
We need example of driving long, random access list from a database —  RecyclerView pulling from Rooms or SqlDelight

**Comment by RotBolt** [comment]
Is this still a valid issue? If yes, I would like to work upon it

**Comment by RotBolt** [comment]
@rjrjr @JustinDSN Is this still a valid issue? If yes, I would like to work upon it

**Comment by RotBolt** [comment]
@zach-klippenstein @rjrjr Is this still a valid issue? If yes, I would like to work upon it

**Comment by rjrjr** [comment]
Yes please! It would need to be a separate module so that none of the existing ones get tied to whatever SqlDelight or whatever. At this point a Compose example might be more useful than RecyclerView or AndroidX Paging.

**Comment by rjrjr** [comment]
@steve-the-edwards or @RBusarow  would probably be a better reviewer / co-designer for it than I would.

**Comment by RotBolt** [comment]
Sure, I will get started with jetpack compose only. @steve-the-edwards @RBusarow let me know if there is anything I need to know apart from the workflow-kotlin docs

**Comment by rjrjr** [comment]
🍿 

It'd be wise to share a quick description of whatever design you're thinking of as a sanity check.

**Comment by RotBolt** [comment]
Sure. I will share by weekend
